html-forms/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>HTML Form</title>
+</head>
+
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="get">
+    <p>Email</p>
+    <input type="text">
+
+    <p>Why is coding cool?</p>
+    <select name="Select an option">
+      <option value="Select an option">Select an option</option>
+      <option value="Interesting">Because it's Interesting</option>
+      <option value="Fun">Because it's fun</option>
+      <option value="Powerful">Because it's Powerful</option>
+    </select>
+
+    <p>Favorite Language</p>
+    <input type="radio" name="HTML" /> HTML
+    <input type="radio" name="CSS" /> CSS
+    <input type="radio" name="Java" /> Javascript
+
+    <p>Would you like to sign up for more information?</p>
+    <input type="checkbox" name="sign" /> Sign me Up!
+
+    <p>Comments</p>
+    <textarea name="comments" cols="40" rows="10"></textarea>
+
+    <br>
```

Comment:
Instead of using `br` elements, you should make the input below this a child of a `div` element.